packages/react-native-bridge/android/settings.gradle

```diff
@@ -1,5 +1,57 @@
-rootProject.name = '@wordpress/react-native-bridge'
+pluginManagement {
+  plugins {
+    id 'com.jfrog.bintray' version '1.8.5'
+  }
+}
 
+rootProject.name = '@wordpress_react-native-bridge'
+
+include ':react-native-bridge'
 include ':@wordpress_react-native-aztec'
 project(':@wordpress_react-native-aztec').projectDir = new File(rootProject.projectDir, '../../react-native-aztec/android')
 
+def publishReactNativeBridgeVersion = properties["publishReactNativeBridgeVersion"]
+gradle.ext.willPublishReactNativeBridgeBinary = publishReactNativeBridgeVersion?.trim() as boolean
+
+// Only add these modules/projects if we are not building to publish a binary.
+// TODO: This is a very fragile check. Since the `settings.gradle` file is part of initialization,
+// it'll not work for even simple tasks such as `./gradlew clean` unless the
+// `publishReactNativeBridgeVersion` property is passed. It's a necessary check for the moment,
+// however by moving the Hermes binary to remote and maybe using dependency substitution for the
+// included projects, we can get rid of it.
```

Comment:
@mokagio @mchowning @mokagio I wanted to let you know about this recent change. I was looking into whether we could run the publish script without the `node_modules` folders (for CI) and we technically can, but it gets very messy because of this check.

I just wanted to note that this is far from ideal, but also not important enough to work on right now, so I opted to just leave a comment here for now.

Reply:
Thanks for the message @oguzkocer.

With your recent work removing this in https://github.com/oguzkocer/gutenberg/pull/3, should we then be able to remove the `node_modules` requirement in CI? Although, that would be a marginal  improvement only, because we'll still need to run `npm install` (`npm ci` to be precise) in order to build the JS bundle, so 🤷‍♂️ ?

Reply:
Unfortunately it's the opposite as `node_modules` is now a hard requirement. Specifically [this assertion](https://github.com/oguzkocer/gutenberg/pull/3/files#diff-e0b35174a44c650c42f00c9b0ac7dc0a73fa7c19a1288f48c44301ff4e22065aR13) will not allow any Gradle task to be run if `node_modules` is missing.

Reply:
Gotcha